external/radiation/radiation/radiation_driver.py

```diff
@@ -196,6 +200,7 @@ def radupdate(
         cline,
         solar_data,
         gas_data,
+        me,
```

Comment:
me! (I assume this name is copied from Fortran?)

Reply:
Ha yeah we can call this `rank` if it's clearer.